eth_test_parser/Cargo.toml

```diff
@@ -15,7 +15,10 @@ eth-trie-utils = { git = "https://github.com/mir-protocol/eth-trie-utils.git", r
 clap = {version = "3.2.17", features = ["derive"] }
 anyhow = { version = "1.0", features = ["backtrace"] }
 log = "0.4.17"
-serde_json = "1.0.85"
+serde_json = "1"
+serde = "1"
+serde_bytes = "0.11"
+serde_with = "2.0"
```

Comment:
Although my initial thought before was not to specify dependency versions down to the patch number for libraries, I recently learned that the standard is to always specify `major.minor.patch` for all library dependencies. There's a post about it [here](https://users.rust-lang.org/t/psa-please-specify-precise-dependency-versions-in-cargo-toml/71277), but we probably should probably do this just to be consistent.